lang/es.json

```diff
@@ -272,13 +272,13 @@
     "ACTOR.TypeToken": "Marcador",
     "ITEM.TypeAbility": "Capacidad",
     "ITEM.TypeAmmo": "Munición",
-    "ITEM.TypePower Shift": "Mejora de Poder",
+    "ITEM.TypePowerShift": "Mejora de Poder",
     "ITEM.TypeSkill": "Habilidad",
-    "ITEM.TypeTeen Ability": "Habilidad (Adolescente)",
-    "ITEM.TypeTeen Armor": "Armadura (Adolescente)",
-    "ITEM.TypeTeen Attack": "Ataque (Adolescente)",
-    "ITEM.TypeTeen Lasting Damage": "Daño Persistente (Adolescente)",
-    "ITEM.TypeTeen Skill": "Habilidad (Adolescente)",
+    "ITEM.TypeTeenAbility": "Habilidad (Adolescente)",
+    "ITEM.TypeTeenArmor": "Armadura (Adolescente)",
+    "ITEM.TypeTeenAttack": "Ataque (Adolescente)",
+    "ITEM.TypeTeenLastingDamage": "Daño Persistente (Adolescente)",
+    "ITEM.TypeTeenSkill": "Habilidad (Adolescente)",
     "CYPHERSYSTEM.SpeedRoll": "Tirada de Velocidad",
     "CYPHERSYSTEM.IntellectRoll": "Tirada de Intelecto",
     "CYPHERSYSTEM.AdditionalPoolRoll": "Tirada de {pool}",
```

Comment:
Please don’t edit the language files (except for en.json) directly. We let Weblate take care of that. I’ve had some data loss after editing a file directly, presumably because Weblate removed all the translated strings after the ones I edited. I wouldn’t want that the others have to do the work twice.